test/integration/requester-spec/headers.test.js

```diff
@@ -0,0 +1,236 @@
+var _ = require('lodash'),
+    sinon = require('sinon'),
+    expect = require('chai').expect;
+
+describe('cache-control and postman-token headers', function () {
+    var testrun,
+        HOST = 'https://www.postman-echo.com/get';
+
+    describe('are not provided', function() {
+        before(function(done) {
+            this.run({
+                collection: {
+                    item: [{
+                        name: 'Cache-Control and postman-token',
+                        request: {
+                            url: HOST
+                        }
+                    }]
+                }
+            }, function (err, results) {
+                testrun = results;
+                done(err);
+            });
+        });
+
+        it('should complete the run', function () {
+            expect(testrun).to.be.ok;
+            sinon.assert.calledOnce(testrun.start);
+            sinon.assert.calledOnce(testrun.done);
+            sinon.assert.calledWith(testrun.done.getCall(0), null);
+
+            sinon.assert.calledOnce(testrun.request);
+            sinon.assert.calledWith(testrun.request.getCall(0), null);
+
+            sinon.assert.calledOnce(testrun.response);
+            sinon.assert.calledWith(testrun.response.getCall(0), null);
+        });
+
+        it('should send request with `Cache-Control: no-cache` header', function () {
+            var request = testrun.request.getCall(0).args[3].toJSON(),
+                response = JSON.parse(testrun.response.getCall(0).args[2].stream.toString());
+
+            expect(request.header).to.deep.include({key: 'Cache-Control', value: 'no-cache'});
+            expect(response.headers).to.have.property('cache-control', 'no-cache');
+        });
+
+        it('should send request with `Postman-Token` header', function () {
+            var request = testrun.request.getCall(0).args[3].toJSON(),
+                response = testrun.response.getCall(0).args[2].stream.toString(),
+
+                regex = /^(Postman-Token: )?[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/mi,
+
+                responseHeader = _.get(JSON.parse(response).headers, 'postman-token'),
+
+                requestHeaders = request.header.filter(function(header) {
+                    return (header.key === 'Postman-Token');
+                });
+
+            expect(regex.test(responseHeader)).to.be.true;
```

Comment:
check for existing of the property(like its done in the first test) and later assert the value to be valid uuid-v4.